conans/client/graph/install_graph.py

```diff
@@ -229,7 +244,7 @@ def _initialize_deps_graph(self, deps_graph):
             else:
                 existing.add(node)
 
-    def install_order(self):
+    def install_order(self, by_levels=True):
```

Comment:
I think that `by_levels` is a little bit confusing. I'm thinking of `flat=True` instead, where `flat = True` means `by_levels = False`. Wdyt?